src/MoonSharp.Interpreter.Tests/EndToEnd/UserDataNestedTypesTests.cs

```diff
@@ -1,8 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using NUnit.Framework;
```

Comment:
Accidental removal appears fixed in later commit: https://github.com/moonsharp-devs/moonsharp/pull/217/commits/ee86463e2ecbc2ff8d1311de4ff806ffc0e8dc38

Reply:
Not a fan of the naming convention here https://github.com/wledfor2 - based on how some other global settings work, and since these are in the same 'fuzzy' class it might be better to have these as an enum/flag construct? Might be my preference speaking here.

Reply:
I am open to suggestions on naming preference. I think an Enum would be preferable, the idea was to make it easier to include future additional fuzzy matching behavior.

Reply:
If you want to resubmit with enums I will try to wrangle out a good name here. If not, let me know and I'll add a refactor of this PR to my backlog.

Reply:
I will work on making the requested changes in the next week (working on finishing another project). Feel free to close this if you need to and I can resubmit it.